test/client/fixtures/sandboxes/cookie-test.js

```diff
@@ -84,7 +84,7 @@ test('get/set', function () {
 asyncTest('path validation', function () {
     var iframe = document.createElement('iframe');
 
-    iframe.src = UrlUtil.getProxyUrl('/data/cookie-sandbox/validation.html');
+    iframe.src = UrlUtil.getProxyUrl(window.QUnitGlobals.getResourceUrl('../../data/cookie-sandbox/validation.html', 'cookie-sandbox/validation.html'));
```

Comment:
yes.
The default is:
`<hostname>/test-resource?filePath=<filePath>&base=<curTestFilePath>`
In my example it will be:
`<hostname>/test-resource/cookie-sandbox/iframe.html?filePath=<filePath>&base=<curTestFilePath>`